nestjs/src/discord-servers/discord-servers.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { DiscordServersService } from './discord-servers.service';
+
+describe('DiscordServersService', () => {
+  let service: DiscordServersService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [DiscordServersService],
+    }).compile();
+
+    service = module.get<DiscordServersService>(DiscordServersService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+});
```

Comment:
there is no need to add such dumb tests. if we have business logic somewhere it should be tested. testing that service is just defined has no value IMO.

Reply:
Agree

Reply:
Deleted this tests at all